src/main/java/com/iridium/iridiumskyblock/commands/RegenCommand.java

```diff
@@ -67,38 +68,45 @@ public boolean execute(User user, Island island, String[] args, IridiumTeams<Isl
             }
         }
 
+        Bukkit.getScheduler().runTaskLater(IridiumSkyblock.getInstance(), () -> {
+            regenerateIsland(player, island, schematicConfig, iridiumTeams);
+        }, 1L);
+
+        return true;
+    }
```

Comment:
Not convinced this is needed, lemme take a look later

Reply:
It is indeed not needed